scripts/automation/release/README.md

```diff
@@ -0,0 +1,36 @@
+Examples of use
```

Comment:
Will these scripts eventually be invoked by the GitHub hooks like you talked about during scrum?